bm-persona/Libraries/LibraryDetailViewController.swift

```diff
@@ -0,0 +1,362 @@
+//
+//  CalendarViewController.swift
+//  bm-persona
+//
+//  Created by Oscar Bjorkman on 2/2/20.
+//  Copyright © 2020 RJ Pimentel. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+fileprivate let kHeaderFont: UIFont = Font.bold(24)
+fileprivate let kCardPadding: UIEdgeInsets = UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16)
+fileprivate let kViewMargin: CGFloat = 16
+
+class LibraryDetailViewController: UIViewController {
+
+    var scrollView: UIScrollView!
+    var libraries: [Library] = []
+    var library : Library?
+    var contentHelper: UIView!
+    var locationManager = CLLocationManager()
+    var userCoords = CLLocationCoordinate2D(latitude: 0.0 , longitude: 0.0 )
+    var distLabel = UILabel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .white
+        
+        //location stuff
+        locationManager.delegate = self
+        
+        // get library data
+        DataManager.shared.fetch(source: LibraryDataSource.self) { libraries in
+            self.libraries = libraries as? [Library] ?? []
+            self.library = libraries[4] as? Library
+            }
```

Comment:
remove? don't need to get library data because the library object is passed in from LibraryViewController